cockatrice/src/settingscache.cpp

```diff
@@ -162,7 +164,16 @@ SettingsCache::SettingsCache()
     if(!QFile(settingsPath+"global.ini").exists())
         translateLegacySettings();
 
+    // updates
+    dummy = QT_TRANSLATE_NOOP("i18n", "Stable releases");
+    dummy = QT_TRANSLATE_NOOP("i18n", "Development snapshots");
```

Comment:
It was used to force qt's l'update to find and extract these translatable strings.
I moved the strings inside the *ReleaseChannel classes and removed this trick.